Contents/mods/Hydrocraft/media/scripts/Carpentry.txt

```diff
@@ -936,7 +942,7 @@ item HCThresher
         Type            = Normal,
         DisplayName             = Thresher,
         icon            = HCThresher,
-    
+		DisplayCategory	= CraftCarp,
```

Comment:
maybe cookRef for cooking refining? The item is nolonger used for carpentry stuff, but processing the ingredients for food, like the cheese vat stuff. @HugoQwerty what do you think?